README.md

```diff
@@ -1,2 +1,270 @@
-# zonos-elements-readme
-Documentation for integrating zonos-elements, no porduction code.
+# Zonos Component UI
+
+## Introduction
+This project offers web components designed primarily for Zonos Checkout, Zonos Hello, and other Zonos products, with a focus on React and Next.js integration. Although not yet tested with other JavaScript libraries and frameworks, the implementation principles for React and Next.js can be adapted for use with other frameworks. For more information about Zonos products, please visit the official [Zonos website](https://zonos.com).
+
+## Installation
+There are two ways to initialize Zonos: via custom integration using a CDN or through npm installation.
+
+**Note: An API key is required to use Zonos Checkout/Hello. Please contact our support team for more information.**
+
+
+- ### CDN
+
+  #### - When to use:
```

Comment:
What do you think about only using the dash for lists?

```suggestion
  #### When to use:
```

Reply:
Yah, it was my original at first, but then seeing that the font size of h4 is too indentical with the content below, that's why I add `-` to make it look a little bit different, not using list because there are already a tons of bullet point below, I think it might be confuse :thinking:. I checked with GPT and it also suggested bullet point. I'll see if GPT has any other way of organizing it. Thanks.